yarn.lock

```diff
@@ -19743,6 +19776,14 @@ react-dom@18.2.0:
     loose-envify "^1.1.0"
     scheduler "^0.23.0"
 
+react-dom@18.3.1:
```

Comment:
we now have 2 react 18 versions in repo - please dedupe this 

![Uploading image.png…]()

Reply:
This is because the `react-18-tests-v8` package uses 18.2.0. Those are v8 tests, so I didn't want to bump it. I changed that now, so I hope it's not going to explode.